### PR TITLE
[RPA-516] Create Templates section.

### PR DESCRIPTION
**Pull request by star-in-the-jar**

### CODE REVIEW

**Comment by Stanlee77** [comment]
Already saw bug with search icon & too big padding of searchbars on small screens. Going to fix it:
image

**Comment by star-in-the-jar** [comment]
The rest of different resolutions:
image

image

image

image

image

image

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { FC, useMemo } from 'react';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import Typography from '#src-landing/components/Typography';
+
+import { categoriesTranslationKeys } from '../TemplatesSection.utils';
+import styles from './Categories.module.scss';
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx:7]: separe groups

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.searchBar {
+    z-index: 10;
+    border-radius: $margin-5;
+    border-width: 3px;
+    border-style: solid;
+    border-color: rgb($text-color-gray, 0.2);
+    background-color: rgb($text-color-gray, 0.05);
+    width: 100%;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss:13]: styles order width, padding to top

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    color: rgb($text-color-gray, 0.4);
+    background-image: url('#public/images/icons/search.svg');
+
+    &:focus {
+        border-color: $palette-accent-primary;
+        color: $palette-accent-primary;
+    }
+    
+    &::-webkit-input-placeholder {
+        color: rgb($text-color-gray, 0.4);
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss]: create new variable in fonts scss called "text-input-placeholder"

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        color: $palette-accent-primary;
+    }
+}
+
+.searchIcon {
+    position: absolute;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    right: 0;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss]: styles order right to the top

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+
+import Image from 'next/image';
+
+import searchIconSrc from '#public/images/icons/search.svg';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import styles from './SearchBar.module.scss';
+import { SearchBarProps } from './SearchBar.types';
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx:9]: separate imports

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+const HighlightedText:FC<HighlightTextProps> = ({
+    textSource,
+    styleClass,
+    highlight,
+}) => {
+    const textToHighlight = replaceRegExpUnallowedChars(highlight);
+    const highlightPattern = createPattern(textToHighlight);
+    const highlightOccurences = useMemo(() => [...textSource.matchAll(highlightPattern)], [textSource, highlightPattern]);
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.tsx]: enter

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.utils.ts [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export const replaceRegExpUnallowedChars = (text: string) => text.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.utils.ts:1]: rename unallowed to special. check if works properly

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
+@import "#src-landing/scss/main.scss";
+
+.root {
+    display: flex;
+    position:relative;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: space after :

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: $margin-2;
+    background-color: $palette-background-primary;
+    box-shadow: $box-shadow-centered;
+    transition: transform 200ms ease-in-out;
+    padding: $padding-4;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: move padding up

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.tsx [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+                        styleClass={styles.textHighlight}
+                        highlight={highlight}
+                    />
+                </Typography>
+            </div>
+            <div className={styles.tileDetails}>
+                <Typography>
+                    {translate('Landing.Templates.TemplatesGrid.TemplateTile.Category')}
+                    <span className={styles.boldText}>
+                        {/* @ts-ignore */}
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.tsx]: remove ts-ignore

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss [hunk 11]
```diff
@@ -0,0 +1,51 @@
+@import "#src-landing/scss/main.scss";
+
+$WIDTH_SCREEN_TILE_SIZE: 400px;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss]: WIDTH -> WIDE

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+    align-items: center;
+    width: 100%;
+}
+
+.templatesGrid {
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax($WIDTH_SCREEN_TILE_SIZE, 1fr));
+    grid-gap: $margin-4;
+    width: 100%;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss:34]: styles order

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.tsx [hunk 13]
```diff
@@ -0,0 +1,39 @@
+import React, { FC, useMemo} from 'react';
+
+import { replaceRegExpUnallowedChars, createPattern } from './HighlightedText.utils';
+
+interface HighlightTextProps {
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/HighlightedText/HighlightedText.tsx]: rename to highlightED...TextProps

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx [hunk 14]
```diff
@@ -0,0 +1,89 @@
+import React, { FC, useState, useMemo, useEffect } from 'react';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import Typography from '#src-landing/components/Typography';
+
+import Categories from './Categories';
+import SearchBar from './SearchBar';
+import styles from './TemplatesSection.module.scss';
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx:9]: separate imports

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+    const filteredTemplates = useMemo(() => templates
+        .filter(template => selectedCategory === 'All' ? true : template.categories.includes(selectedCategory))
+        .filter(template => template.title.toLowerCase().includes(searchValue.toLowerCase()) 
+            || template.description.toLowerCase().includes(searchValue.toLowerCase())), [selectedCategory, searchValue]);
+
+    const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setSearchValue(event.target.value);
+    };
+
+    const handleCategoryChange= (categoryKey: string) => {
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx]: space before =

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.tsx [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+    categories,
+    integrations,
+    highlight,
+}) => {
+    const { translate } = useTranslations();
+
+    const translatedCategories = categories.length > 0 
+        // @ts-ignore
+        ? categories.map((categoryKey) => translate(`Landing.Templates.CategoriesBar.Category.${categoryKey}`)).join(', ')
+        : translate('Landing.Templates.CategoriesBar.Category.None');
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.tsx]: if no category just don't display categories at all

**Comment by star-in-the-jar** [comment]
and mobile resolutions:
image

image

**Review by star-in-the-jar — COMMENTED** [review]
Self reviewed

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+    flex-wrap: wrap;
+    font-size: $font-size-h6;
+}
+
+.category {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 1.5rem;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss]: padding vars here

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+
+const Categories: FC<CategoriesProps> = ({ 
+    selectedCategoryKey, 
+    handleCategoryChange,
+}) => {
+    const { translate } = useTranslations();
+
+    const categoriesList = useMemo(() => categoriesTranslationKeys.map((categoryKey) => {
+        // @ts-ignore
+        const categoryName = translate(`Landing.Templates.CategoriesBar.Category.${categoryKey}`);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx]: if u are absolutely sure just do assertion here istead of ts ignore. But personaly i would like to see ts guard checking if this translation key is `keyof typeof temaplatesTranslations`, if not provide fallback value

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+
+const SearchBar:FC<SearchBarProps> = ({
+    searchValue,
+    handleSearch,
+}) => {
+    const { translate } = useTranslations();
+
+    return (
+        <div className={styles.wrapper}>
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: this class should be root

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss [hunk 20]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.boldText {
+    font-weight: $font-weight-semi-bold;
+}
+
+.integrations {
+    display: grid;
+    align-items: center;
+    grid-template-columns: repeat(auto-fill, minmax(100px, 2fr));
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: move this 100px to var

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.textHighlight {
+    color: $palette-accent-primary;
+    font-weight: $font-weight-bold;
+}
+
+@media (max-width: $breakpoint-sm) {
+    .integrations {
+        grid-template-columns: repeat(auto-fill, minmax(60px, 1fr));
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: same here

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+
+    const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setSearchValue(event.target.value);
+    };
+
+    const handleCategoryChange = (categoryKey: string) => {
+        setSelectedCategory(categoryKey);
+    };
+    
+    const handleResize = () => {
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx:63]: Utalize CSS variables and media queries, look at the carousel for reference. Timeout is a bit of an hack in my opinion, but clever one

**Review by FranekMarciniak — CHANGES_REQUESTED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss (hunk 17 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx (hunk 19 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss (hunk 20 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss (hunk 21 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx (hunk 22 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx:63]: thanks, took the carousel for an example. Workin and lookin' good

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx:63]: done

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx:63]: done, a little bit in my wya. Hope its all right

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx (hunk 18 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss [hunk 23]
```diff
@@ -0,0 +1,60 @@
...
+    justify-content: space-between;
+    padding: $padding-2 $padding-3;
+    color: $text-color-dark;
+    text-decoration: none;
+    transition: font-weight 300ms ease;
+
+    // to stop from flicking on focus/hover -> font-weight change
+    &::before {
+        content: attr(data-text);
+        content: attr(data-text) / "";
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss]: .

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx [hunk 24]
```diff
@@ -0,0 +1,27 @@
...
+    handleSearch,
+}) => {
+    const { translate } = useTranslations();
+
+    return (
+        <div className={styles.root}>
+            <div className={styles.searchIcon}>
+                <Image src={searchIconSrc} alt="search" />
+            </div>
+            <input className={styles.searchBar} onChange={handleSearch} value={searchValue} placeholder={translate('Landing.Templates.SearchBar.Placeholder')} />
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: move to seperate lines

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+const SearchBar:FC<SearchBarProps> = ({
+    searchValue,
+    handleSearch,
+}) => {
+    const { translate } = useTranslations();
+
+    return (
+        <div className={styles.root}>
+            <div className={styles.searchIcon}>
+                <Image src={searchIconSrc} alt="search" />
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: there should be translations

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss (hunk 8 above)
Comment (FranekMarciniak) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: .

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss (hunk 9 above)
Comment (FranekMarciniak) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplateTile/TemplateTile.module.scss]: .

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss (hunk 11 above)
Comment (FranekMarciniak) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss]: .

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss [hunk 26]
```diff
@@ -0,0 +1,59 @@
...
+
+$WIDTH_SCREEN_TILE_SIZE: 400px;
+$NARROW_SCREEN_TILE_SIZE: 250px;
+
+.root {
+    composes: root from '#src-landing/components/SectionContent/SectionContent.module.scss';
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    --tiles-number: 6;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss:11]: <3

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+    const { translate } = useTranslations();
+    const [selectedCategory, setSelectedCategory] = useState<string>(categoriesNames.CATEGORY_ALL);
+    const [searchValue, setSearchValue] = useState<string>('');
+    const gridRef = useRef<HTMLDivElement>(null);
+    const [maxDisplayedTiles, setMaxDisplayedTiles] = useState<number>(6);
+    
+    const filteredTemplates = useMemo(() => templates
+        .filter(template => selectedCategory === 'All' ? true : template.categories.includes(selectedCategory))
+        .filter(template => template.title.toLowerCase().includes(searchValue.toLowerCase()) 
+        || template.description.toLowerCase().includes(searchValue.toLowerCase())), [selectedCategory, searchValue]);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx]: - move dependencies array to new line
- move template.$VAR.toLowerCase().includes(searchValue.toLowerCase())) to a function

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx [hunk 28]
```diff
@@ -0,0 +1,88 @@
...
+    
+    const tiles = useMemo(() => filteredTemplates.slice(0, maxDisplayedTiles).map((template) => (
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx]: array methods on new line

**Review by FranekMarciniak — CHANGES_REQUESTED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss (hunk 23 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.module.scss]: removed

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx (hunk 24 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss (hunk 26 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss:11]: :D

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx (hunk 28 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx (hunk 27 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx (hunk 25 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.tsx]: done

**Review by star-in-the-jar — COMMENTED** [review]
Self reviewed.

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss [hunk 29]
```diff
@@ -0,0 +1,59 @@
...
+        padding-top: $padding-4;
+        padding-bottom: $padding-4;
+        --tiles-number: 2;
+    }
+}
+
+@media (max-width: $breakpoint-xs) {
+    .templatesGrid {
+        padding-left: $padding-0;
+        padding-right: $padding-0;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss:57]: padding inline or padding: inherit/initial (i dont remember which one) $padding-0; same above

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss [hunk 30]
```diff
@@ -0,0 +1,63 @@
...
+        width: $margin-4;
+        height: $margin-4;
+    }
+}
+
+@media (max-width: $breakpoint-sm) {
+    .wrapper {
+        width: 100%;
+        margin-top: $margin-3;
+        margin-bottom: $margin-3;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss]: can be styled with one margin statment

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss [hunk 31]
```diff
@@ -0,0 +1,63 @@
+@import "#src-landing/scss/main.scss";
+
+.root {
+    position: relative;
+    width: 70%;
+    margin-top: $margin-8;
+    margin-bottom: $margin-8;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/SearchBar/SearchBar.module.scss:7]: same here

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss (hunk 29 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/TemplatesSection.module.scss:57]: done

**Review by star-in-the-jar — COMMENTED** [review]
Self reviewed.

**Review by FranekMarciniak — APPROVED** [review]

**Pull request merged**